src/bioregistry/data/bioregistry.json

```diff
@@ -22211,6 +22211,58 @@
     "name": "Crates.io",
     "uri_format": "https://crates.io/crates/$1"
   },
+  "creativebiolabs.antibody": {
+    "contact": {
+      "email": "info@creative-biolabs.com",
+      "name": "Creative Biolabs Team"
+    },
+    "contributor": {
+      "email": "placeholder@example.com",
+      "github": "garybrandam",
+      "name": "Gary",
+      "orcid": "0000-0000-0000-0000"
```

Comment:
@garybrandam we require ORCIDs for contributions and can't proceed with merging the additions unless you can provide an ORCID. We also generally require emails for contributors. Please let us know and we can add it!